release/src/main/scripts/build_release_candidate.sh

```diff
@@ -96,20 +102,42 @@ if [[ $confirmation = "y" ]]; then
   git checkout ${RELEASE_BRANCH}
   RELEASE_COMMIT=$(git rev-parse --verify ${RELEASE_BRANCH})
 
-  echo "-------------Building Java Artifacts with Gradle-------------"
-  git config credential.helper store
+  echo "[Current Step]: Build and stage java artifacts"
+  echo "Do you want to proceed (skip if retrying staging step)? [y|N]"
+  read confirmation
+  if [[ $confirmation = "y" ]]; then
```

Comment:
Can we also control these with env vars? The goal IMO is complete non-interactivity / scriptability.